### ISSUE TITLE
aria-live not announced when focused on role="application"

### ISSUE THREAD

**Issue by gdkraus** [FreedomScientific/VFO-standards-support#110]
### Summary

We noticed a change in behavior with role="application" and aria-live that we believe is tied to the Chrome 69 release. This problem arises with forms mode being set to auto. If you have a role="application" area, but there are no focusable elements within the application (you stay focused on the application itself as a whole), if there are changes to an aria-live region outside of the application, then JAWS is not announcing the changes to the aria-live region. We discovered JAWS would announce changes to the live region if JAWS automatically went into forms mode when it encountered the role="application". The difference we noticed in Chrome 69 was that if there were no focusable items with in the role="application", JAWS would not go into forms mode, but if there are any focusable items in the application, JAWS would enter forms mode.

To see this in action, go to 

https://codepen.io/gdkraus-sas/full/LJOzXb/

The first role="application" does not have any focusable child elements. While focused on it, pressing any key will update the time in the live region, but the live region is not announced. Toggling the virtual cursor has no impact on the behavior of announcing the live region.

The second role="application" has an input as a child element. When focused on this application, pressing any key will update a live region with the current time and the live region gets announced (except for the first time, but we can set that issue aside for now).

Further, if you turn forms mode to off, if you tab through the page to the second application, forms mode turns on, then tab back to the first application, forms mode is still on, and the live region announces correctly. However, just tabbing to the first application, there does not appear to be a way to get the live region to announce.

The fundamental problem seems to be forms mode must be on while focused on an application for the live region to announce correctly, and simply toggling the virtual cursor on and off is not enough to solve the problem.

NVDA still announces changes for both application regions.

### Expected result

The live region should announce changes

### Actual result

The live region did not announce changes when application had no focusable items

### Example

https://codepen.io/gdkraus-sas/full/LJOzXb/

#### JAWS version and build number

JAWS 2018.1808.10 ILM
 
#### Operating System and version
Windows 10 Enterprise

#### Browser and version:
Google Chrome 69.0.3497.81

**Comment by gdkraus** [comment]
Note: JAWS 18 behaves as expected, going into forms mode when focused on either role="application", and thus announcing the live region when it changes.

**Comment by JAWS-test** [comment]
The problem no longer occurs with JAWS 2019 (in Chrome and Firefox; not testable in IE because the page is not displayed by IE)
